py2many/cli.py

```diff
@@ -73,16 +73,19 @@ def core_transformers(tree):
     return tree, infer_meta
 
 
-def transpile(filename, source, transpiler, rewriters, transformers, post_rewriters):
+def transpile(filenames, sources, transpiler, rewriters, transformers, post_rewriters):
```

Comment:
IMO `_` prefix this, as its signature is likely to keep changing in backwards-incompatible ways.

Reply:
I renamed to `_transpile_one` and `_process_one` and changed the signature of this function with types so hopefully it's more stable going forward.

Reply:
It wont be stable :P

Reply:
Renamed. I think you're trying to say `main()` is the only stable interface.

Reply:
ya, I expect all of the guts of `cli`, except for `main`, will be moved to different modules, and become more structured.

This PR is the perfect opportunity to make it private, as it was already fiddling with all of the relevant uses of the function.